ci: update build and preview workflows

4rthurLegrand opened the pull request.

tbouffard commented:
⚠️ bcd 3.6 requires bonita@2022.1. Yes, I know this is a mess 👿 
```
 [14:40:13.758] ERROR (asciidoctor): target of xref not found: 2022.1@bonita::workspaces-and-repositories.adoc
    file: modules/ROOT/pages/livingapp_build.adoc:5
    source: https://github.com/bonitasoft/bonita-continuous-delivery-doc.git (branch: 3.6)
[14:40:13.759] ERROR (asciidoctor): target of xref not found: 2022.1@bonita::workspaces-and-repositories.adoc#_use_a_shared_project
    file: modules/ROOT/pages/livingapp_build.adoc:14
    source: https://github.com/bonitasoft/bonita-continuous-delivery-doc.git (branch: 3.6)
[14:40:13.763] ERROR (asciidoctor): target of xref not found: 2022.1@bonita::api-extensions.adoc
    file: modules/ROOT/pages/livingapp_build.adoc:41
    source: https://github.com/bonitasoft/bonita-continuous-delivery-doc.git (branch: 3.6)
[14:40:13.766] ERROR (asciidoctor): target of xref not found: 2022.1@bonita::workspaces-and-repositories.adoc
    file: modules/ROOT/pages/livingapp_build.adoc:81
    source: https://github.com/bonitasoft/bonita-continuous-delivery-doc.git (branch: 3.6)
[14:40:13.783] ERROR (asciidoctor): target of xref not found: 2022.1@bonita::share-a-repository-on-github.adoc
    file: modules/ROOT/pages/livingapp_build_and_deploy.adoc:34
    source: https://github.com/bonitasoft/bonita-continuous-delivery-doc.git (branch: 3.6)
```

4rthurLegrand commented:
Thank you for your comment @tbouffard :)
I think that I understood how it works and how to perform the tests.
I modified BONITA_BRANCH value to '2022.2', but I shouldn't. I reverted it to the initial value of '2022.1' and now it seems better!

tbouffard reviewed (approved):
LGTM, nice work!
